addons/account/models/account.py

```diff
@@ -702,6 +702,18 @@ def copy(self, default=None):
         default = dict(default or {}, name=_("%s (Copy)") % self.name)
         return super(AccountTax, self).copy(default=default)
 
+    @api.multi
+    def name_get(self):
+        if not self._context.get('include_taxes_type'):
+            return super(AccountTax, self).name_get()
+        result = []
+        type_tax_use_selection = dict(self._fields['type_tax_use'].selection)
```

Comment:
Yeah. Sadly it's quite ugly: rather than access `selection` you need to call `_selection_description(self.env)`.

That will also properly handle the case where `selection` is a function though I guess that's not the case for type_tax_use.